src/test/java/com/google/api/codegen/testdata/nodejs_doc_version_index_library.baseline

```diff
@@ -0,0 +1,19 @@
+============== file: src/index.js ==============
+/*
+ * Copyright 2016 Google Inc. All rights reserved.
```

Comment:
Lol you are right. This is a from a shared component so I prefer to fix in a separate PR. https://github.com/googleapis/toolkit/issues/897